R/R.r

```diff
@@ -1,6 +1,6 @@
 # R("-e 'str(as.list(Sys.getenv()))' --slave")
 R <- function(options, path = tempdir(), env_vars = NULL, ...) {
-  options <- paste("--vanilla", options)
+  options <- paste("--no-site-file", "--no-environ", options)
```

Comment:
Are you sure? `R --help` gives me :  `--vanilla	Combine --no-save,
--no-restore, --no-site-file, --no-init-file and --no-environ`

On 4/22/15, Kirill Müller <notifications@github.com> wrote:
>> @@ -1,6 +1,6 @@
>>  # R("-e 'str(as.list(Sys.getenv()))' --slave")
>>  R <- function(options, path = tempdir(), env_vars = NULL, ...) {
>> -  options <- paste("--vanilla", options)
>> +  options <- paste("--no-site-file", "--no-environ", options)
>
> Hm... `--vanilla` doesn't imply `--no-save`, but I agree it's better to add
> it anyway.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/hadley/devtools/pull/767/files#r28900125


-- 
http://had.co.nz/